coralproject/talk
# [CORL-1349] Sarcasm Font Fix

## What does this PR do?

Ensure that we only use the feature flag for the sarcasm rich text editor feature.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



Enable the `RTE_SARCASM` feature flag, notice that you can compose, submit, and view sarcasm fonts.
